### ISSUE TITLE
Deleting a bullet still remains indented

### ISSUE THREAD

**Issue by garyking** [neilsustc/vscode-markdown#244]
#### What is the problem?

In the release of July 22, 2018, an indent still remains after deleting a bullet.

#### How can I reproduce it?

Type `- ` then delete it. The cursor will be indented two spaces, rather than indented zero spaces, like it was before.

#### Is there any error message in the console?

No

---

Please restore behavior to before, or add an option to do so. Thanks!

**Comment by neilsustc** [comment]
Yes, this is an intentional change. I will introduce a new option for it.

**Comment by neilsustc** [comment]
One more question, are you trying to delete the list marker and then press `enter` to start a new line?
You can directly press `enter`, empty list item will be removed automatically.

**Comment by garyking** [comment]
Okay I'll keep that in mind
